src/System.Runtime/tests/System/EnumTests.netcoreapp.cs

```diff
@@ -74,10 +75,14 @@ public static void Parse_Invalid_NetCoreApp11(Type enumType, string value, bool
 
         public static IEnumerable<object[]> UnsupportedEnumType_TestData()
         {
+#if netcoreapp
```

Comment:
For these ifs, can you add a comment with a TODO for the issue number so we follow-up to remove them?